### ISSUE TITLE
suggestion: check for "Linux" not "Ubuntu"?

### ISSUE THREAD

**Issue by andreww35t** [umaumax/clip-share#1]
Suggested diff to clip-share in ref 5ba2b95e9245628e1b49d92c72dc1e4f6870b21c :

```
34c34
< if [[ $(uname -a) =~ "Ubuntu" ]]; then
---
> if [[ $(uname -a) =~ "Linux" ]]; then
```

'Just a suggestion.  clip-share is a great little tool.  And it works well on non-Ubuntu distros too!  I use it to sync my clipboard between my (RH-based) Linux workstation and the KVM virtual machines I run on it.  If not, I can always change it manually.  Thanks for sharing this @umaumax !

**Comment by umaumax** [comment]
Thank you for the suggestion @andreww35t  !
I have tried to fix it in this commit 20264d2 .
How about this?

**Comment by andreww35t** [comment]
@umaumax looks good!  Thanks   :saluting_face:
